examples/platform/nrf528xx/app/support/CXXExceptionStubs.cpp

```diff
@@ -63,18 +63,18 @@ void __cxa_rethrow()
     abort();
 }
 
-void * __cxa_begin_catch(void *)
-{
-    abort();
-}
+// void * __cxa_begin_catch(void *)
```

Comment:
Why commented out?

Reply:
```
.../chip/chip-demo-202008/third_party/pigweed/repo/.environment/cipd/pigweed/bin/../lib/gcc/arm-none-eabi/9.3.1/../../../../arm-none-eabi/lib/thumb/v7e-m+fp/hard/libstdc++_nano.a(eh_catch.o): in function `__cxa_get_exception_ptr':
eh_catch.cc:(.text.__cxa_get_exception_ptr+0x0): multiple definition of `__cxa_get_exception_ptr'; obj/third_party/connectedhomeip/examples/platform/nrf528xx/app/support/chip-nrf52840-lock-example.CXXExceptionStubs.cpp.o:.../source/chip/chip-nrf5-qrcode/examples/lock-app/nrf5/out/debug/../../third_party/connectedhomeip/examples/platform/nrf528xx/app/support/CXXExceptionStubs.cpp:77: first defined here
```

This is caused by STL exceptions introduced by setup payload, which is also the reason of the huge size increase, this is expected to be removed after we have another setup payload implemtation without stl.

Reply:
Should we delete instead of commenting out. I generally would try to avoid commented out code.